src/ui/ImageURLEditor.tsx

```diff
@@ -111,6 +119,15 @@ class ImageURLEditor extends React.Component<Props, State> {
         )
     }
 
+    _onAltTextChange = (e: React.SyntheticEvent<HTMLInputElement>) => {
+        // @ts-ignore
+        const alt = e.target.value
```

Comment:
why `@ts-ignore` ?

I'm also not a huge fan of the `_` prefix on "private" methods, that was also a flow thing I think?  For TS it's better to mark them as private if you really don't want them called from outside the class, but really IMO that's really not needed here either, this could just be `onAltTextChange`